server/test/controllers/document.specs.js

```diff
@@ -0,0 +1,373 @@
+
+import { expect } from 'chai';
+import supertest from 'supertest';
+import 'babel-register';
+import auth from '../../helper/auth';
+import mockData from '../mockData/mockData';
+import app from '../../../build/server';
+
+const superAdminToken = auth.setUserToken(mockData.superAdmin);
+const badToken = mockData.badToken;
+
+const userToken = auth.setUserToken(mockData.user);
+const request = supertest(app);
+
+describe(' Document', () => {
+  describe('create documents function', () => {
+    it('should return a message document created successfully', (done) => {
```

Comment:
This would communicate better to say - "Should return a message message when a document is created successfully"